_locales/cs/maqueenPlusV2-jsdoc-strings.json

```
@@ -1,23 +1,52 @@
+    "maqueenPlusV2.showColor|param|rgb": "Barva ve formátu RGB, např.: colors(NeoPixelColors.Violet)",
+    "maqueenPlusV2.setIndexColor": "Nastaví barvu vybrané posloupnosti RGB LED. Hodnota parametru index určuje konkrétní LED nebo rozsah od-do vytvořený pomocí bloku \"pozice RGB LED od do\".",
+    "maqueenPlusV2.setIndexColor|param|index": "číslo určující konkrétní LED, nebo rozsah LED vytvořený pomocí funkce ledRange.",
+    "maqueenPlusV2.setIndexColor|param|rgb": "Barva ve formátu RGB, např.: colors(NeoPixelColors.Violet)",
+    "maqueenPlusV2.colors": "Vrátí RGB hodnotu zadané barvy z předdefinované palety běžných barev.",
+    "maqueenPlusV2.colors|param|color": "Barva z NeoPixelColors, např.: NeoPixelColors.Violet",
+    "maqueenPlusV2.ledRange": "Vytvoří index adresující posloupnost RGB LED (viz blok \"nastav barvu RGB LED na pozici\").",
+    "maqueenPlusV2.ledRange|param|from": "Pozice první RGB LED od 0 do 3, např.: 1",
+    "maqueenPlusV2.ledRange|param|to": "Pozice poslední RGB LED od 1 do 4, např.: 2",
     "maqueenPlusV2.rgb": "Blok umožňuje definici vlastní barvy pomocí složek červená, zelená a modrá.",
+    "maqueenPlusV2.rgb|param|r": "Hodnota červené složky barvy v rozsahu od 0 do 255, např.: 100",
+    "maqueenPlusV2.rgb|param|g": "Hodnota zelené složky barvy v rozsahu od 0 do 255, např.: 100",
+    "maqueenPlusV2.rgb|param|b": "Hodnota modré složky barvy v rozsahu od 0 do 255, např.: 100",
     "maqueenPlusV2.ledRainbow": "Postupně zobrazí barvy duhy na všech RGB LED.",
+    "maqueenPlusV2.ledRainbow|param|startHue": "Počáteční barevný odstín od 0 do 360, např.: 1",
+    "maqueenPlusV2.ledRainbow|param|endHue": "Konečný barevný odstín od 0 do 360, např.: 360",
     "maqueenPlusV2.ledBlank": "Zhasne všechny RGB LED.",
     "maqueenPlusV2.I2CInit": "Blok spustí inicializaci I2C sběrnice a čeká, dokud úspěšně neproběhne. Tento blok je zapotřebí použít pouze jednou, a to při startu programu.",
+    "maqueenPlusV2.MyEnumMotor": "Možnosti pro výběr motoru.",
+    "maqueenPlusV2.MyEnumDir": "Možnosti pro výběr směru otáček.",
+    "maqueenPlusV2.MyEnumLed": "Možnosti pro výběr předních LED (světlometů).",
+    "maqueenPlusV2.MyEnumSwitch": "Možnosti pro výběr stavu předních LED (světlometů).",
+    "maqueenPlusV2.MyEnumLineSensor": "Možnosti pro výběr senzoru čáry.",
     "maqueenPlusV2.NeoPixelColors": "Definice nejběžnějších barev.",
     "maqueenPlusV2.intersectionDetecting": "Návratová hodnota: 1 představuje křižovatku, 2 představuje křižovatku pro odbočení vlevo a vpravo (T-křižovatku), 3 představuje křižovatku pro odbočení vlevo a jízdu rovně, 4 představuje křižovatku pro odbočení vpravo a jízdu rovně.",
     "maqueenPlusV2.readRealTimeSpeed": "Návratová hodnota jednotky: centimetry za sekundu (cm/s)",
-    "maqueenPlusV2": "Rozšíření pro DFRobot Maqueen Plus v2"
+    "maqueenPlusV2": "Rozšíření pro DFRobot Maqueen Plus v2",
+    "IR.IR_read": "Přečte hodnotu z infra přijímače.",
```

How about calling it with a full name rather than an abbreviation?
```suggestion
    "IR.IR_read": "Přečte hodnotu z infračerveného přijímače.",
```

---

Hmm, why not. It's description text, so it should be descriptive :-)